runner/run.go

```diff
@@ -35,6 +36,7 @@ func Run(include string, exclude string, showTime bool, output bool, ftwtests []
 			// if we received a particular testid, skip until we find it
 			if needToSkipTest(include, exclude, t.TestTitle, tests.Meta.Enabled) {
 				addResultToStats(Skipped, t.TestTitle, &stats)
+				log.Info().Msgf("Skipping test %s", t.TestTitle)
```

Comment:
The problem here is that we cannot control if this is logged in quiet mode.

Reply:
I've changed it to respect quiet mode. Better?